lib/screens/onboarding/helpers.dart

```diff
@@ -1,7 +1,8 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:intro_views_flutter/Models/page_view_model.dart';
 
+import '../../main.dart';
```

Comment:
Any better solution than importing main file inside of helper for accessing global variable?
Maybe we can make Config to return singleton that we can import and access here?